src/android_components.py

```diff
@@ -396,35 +392,23 @@ def _update_application_services(
 #
 
 
-def update_main(ac_repo, fenix_repo, author, debug, dry_run):
-    try:
-        _update_application_services(ac_repo, fenix_repo, None, author, debug, dry_run)
-    except Exception as e:
-        print(
-            f"{ts()} Exception while updating A-S on A-C {ac_repo.full_name}:main: {str(e)}"
-        )
-
-    try:
-        _update_geckoview(ac_repo, fenix_repo, "main", author, debug, dry_run)
-    except Exception as e:
-        print(
-            f"{ts()} Exception while updating GeckoView on A-C {ac_repo.full_name}:main: {str(e)}"
-        )
+def update_main(ac_repo, author, dry_run):
+    branch_name = "main"
+    current_ac_version = get_current_ac_version(ac_repo, branch_name)
+    ac_major_version = int(major_as_version_from_version(current_ac_version))
```

Comment:
```suggestion
    ac_major_version = int(major_ac_version_from_version(current_ac_version))
```
👀 They do the same thing, but I switched it for correctness.